README.md

```diff
@@ -40,364 +43,346 @@ some kind of discriminant:
+- [`const enum`](#const-enum)
+- [`Symbol` values](#symbol-values)
+- [`BigInt` values](#bigint-values)
+- [`export default`](#export-default)
+
+## Auto-Initializers
 
-To make it easier to work with enums, an `Enum` object is added to the global scope, with the following
-methods:
-
-- `Enum.keys(E)` - Returns an `Iterator` for the member names in the \[\[EnumMembers]] internal 
-  slot of `E`.
-- `Enum.values(E)` - Returns an `Iterator` for the value on `E` of each member in the 
-  \[\[EnumMembers]] internal slot of `E`.
-- `Enum.entries(E)` - Returns an `Iterator` for each member in the \[\[EnumMembers]] internal
-  slot of `E`, where each result is two-element array containing the enum member name at index 0 
-  and the enum member value at index 1.
-- `Enum.has(E, key)` - Returns `true` if the the \[\[EnumMembers]] internal slot of `E` contains `key`.
-- `Enum.hasValue(E, value)` - Returns `true` if the \[\[EnumMembers]] internal slot of `E` contains a
-  member whose value on `E` corresponds to `value`.
-- `Enum.getName(E, value)` - Gets the first name in the \[\[EnumMembers]] internal slot of `E` whose
-  value on `E` corresponds to `value`.
-- `Enum.format(E, value)` - Calls the `@@formatEnum` method of `E` with argument `value`.
-- `Enum.parse(E, value)` - Calls the `@@parseEnum` method of `E` with argument `value`.
-- `Enum.create(members)` - Creates an _enum object_ using the property keys and values of `members`
-  as the _enum members_ for the new enum.
-- `Enum.flags(descriptor)` - A built-in decorator that modifies the _enum object_ in the following ways:
-  - The auto-increment behavior is changed to shift the current auto-increment value left by 1. 
-  - The `@@parseEnum` method is modified to parse a comma-separated string and OR the resulting values
-    together. If no corresponding name can be found and the name can be successfully coerced to a number,
-    that number is OR'ed with the result.
-  - The `@@formatEnum` method is modified to convert a bitwise combination of flag values into a comma
-    separated string of corresponding names. If no corresponding name can be found, the SV of the 
-    bits is appended to the string.
+TypeScript's `enum` supports auto-initialization of enum members:
 
 ```ts
-let Enum: {
-  keys(E: object): IterableIterator<string | symbol>;
-  values(E: object): IterableIterator<any>;
-  entries(E: object): IterableIterator<[string | symbol, any]>;
-  has(E: object, key: string | symbol): boolean;
-  hasValue(E: object, value: any): boolean;
-  getName(E: object, value: any): string | undefined;
-  format(E: object, value: any): string | symbol | undefined;
-  parse(E: object, value: string): any;
-  create(members: object): object;
-  flags(descriptor: EnumDescriptor): EnumDescriptor;
-};
+enum Numbers {
+  zero, // 0
+  one,  // 1
+  two   // 2
+}
 ```
 
-<!--
-# Grammar
+However, this behavior is contentious amongst some TC39 delegates and has been removed from this proposal. TypeScript
```

Comment:
Can you link the decision or an issue?
Was it a vote by majority?

Reply:
This is from a number of offline conversations with various delegates over the years, though I believe it did come up in the tc39-delegates matrix chat during the last plenary or back when @Jack-Works presented his `enum` proposal several years ago. I'll see if I can find some links.

Reply:
One of the conversations starts https://matrixlogs.bakkot.com/TC39_Delegates/2025-02-20#L634 (just before UTC midnight) and then continues https://matrixlogs.bakkot.com/TC39_Delegates/2025-02-21#L0

Reply:
It might be worth clarifying in the document that it wasn't considered a good default, because of versioning concerns.